lib/tasks/spin_up.rake

```diff
@@ -32,28 +42,26 @@ class SpinUpHelper
   def setup_secrets_config
     filename = "config/secrets.yml"
     yaml = YAML.load_file(filename)
-    yaml["development"]["secret_key_base"] = SecureRandom.hex(128)
-    yaml["test"]["secret_key_base"] = SecureRandom.hex(128)
+    yaml["development"]["secret_key_base"] = SecureRandom.hex(64)
+    yaml["test"]["secret_key_base"] = SecureRandom.hex(64)
     write_yaml(yaml, filename)
   end
 
   def setup_database
-    command = "rake db:create db:migrate && rake db:seed"
+    if @env == :development
+      command = "rake db:create db:migrate && rake db:seed"
+    elsif @env == :test
+      command = "RAILS_ENV=test rake db:drop db:create db:migrate"
+    else
+      raise "unknown environment"
```

Comment:
Would like this to have basic DB spin as the fall back, I have to spin up up staging and production.